run_tests.sh

```diff
@@ -7,6 +7,18 @@
 # author: andrew schmitt
 # since 7/9/2018
 
+source snapCloud/.env
+export PGDATA=$PG_DATA_DIR
```

Comment:
lol, this is stupid that I defined a similar but different variable in the snapCloud Procfile, but oh wells for now.